python/cucim/src/cucim/skimage/util/__init__.py

```diff
@@ -20,14 +22,15 @@
 from .shape import view_as_blocks, view_as_windows
 
 __all__ = [
-    "img_as_float32",
+    "compare_images" "img_as_float32",
```

Comment:
There is a missing comma between "compare_images" and "img_as_float32" in the `__all__` list.

Reply:
thanks, fixed